local_planner/src/nodes/local_planner.h

```diff
@@ -183,11 +184,13 @@ class LocalPlanner {
   void updateObstacleDistanceMsg(Histogram hist);
   void updateObstacleDistanceMsg();
   void create2DObstacleRepresentation(const bool send_to_fcu);
+  sensor_msgs::Image generateHistogramImage(Histogram& histogram);
 
  public:
   double h_FOV_ = 59.0;
   double v_FOV_ = 46.0;
   Box histogram_box_;
+  sensor_msgs::Image histogram_image_;
```

Comment:
it works the other way round. The main node queries for the image instead of the planner sends it out. Therefore it has to be stored until it is requested